lib/pleaserun/detector.rb

```diff
@@ -25,14 +25,19 @@ def detect_platform
 
   def detect_systemd
     # Expect a certain directory
-    return false unless File.directory?("/lib/systemd/system")
+    return false unless File.directory?("/lib/systemd/system") || File.directory?("/usr/lib/systemd/system")
```

Comment:
I think we can simplify by just checking for /usr/lib/systemd/system, forgetting about /lib/systemd/system. Thoughts?